airflow/contrib/operators/kubernetes_pod_operator.py

```diff
@@ -98,6 +98,9 @@ class KubernetesPodOperator(BaseOperator):
     :type pod_runtime_info_envs: list[PodRuntimeEnv]
     :param dnspolicy: Specify a dnspolicy for the pod
     :type dnspolicy: str
+    :param annotations: non-identifying metadata you can attach to the Pod.
+                        Can be a large range of data, and can include characters that are not permitted by labels.
+    :type annotations: dict
```

Comment:
Could you have the position match the order of arguments to `__init__` (so before `:param resources:`) please?

Reply:
With pleasure.